core/src/test/scala/com/microsoft/azure/synapse/ml/logging/FeatureUsagePayloadTest.scala

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.azure.synapse.ml.logging
+
+import com.microsoft.azure.synapse.ml.core.test.base.TestBase
+import com.microsoft.azure.synapse.ml.logging.Usage.{FeatureSynapseML, FeatureActivityTransform,
+  FeatureActivityTrain, FeatureActivityFit}
+
+class FeatureUsagePayloadTest extends TestBase {
```

Comment:
These tests aren’t really doing much right just checking that the members you have are set the way you set them right?

Reply:
In a way you are right. However, this is the only way to enforce that someone does not inadvertently send some wrong feature and activity names. I would keep this.